docs/react-integration.asciidoc

```diff
@@ -0,0 +1,97 @@
+[[react-integration]]
+=== React integration
+
+This document covers how to use React integration with Real User Monitoring JavaScript agent.
+Please see our <<getting-started, Getting started guide>> for configuring the Real User Monitoring agent.
+
+[[installing-react-integration]]
+==== Installing Elastic APM React package
+
+Install the `@elastic/apm-rum-react` package as a dependency to your application:
+
+[source,bash]
+----
+npm install @elastic/apm-rum-react --save
+----
+
+==== Instrumenting your React application
```

Comment:
Let's add float tags to these headers. They'll render correctly now, but if we add anchors later on they will break 

```suggestion
[float]
==== Instrumenting your React application
```